feature/experimental/system/health/tests/system_generic_health_check/README.md

```diff
@@ -0,0 +1,84 @@
+# Health-1.1: Generic Health Check
+
+## Summary
+
+Generic Health Check
+
+## Procedure
+
+*   Capture the generic health check of the DUT, used modularly in pre/post and during various different tests:
+    *   No system/kernel/process/component coredumps
+    *   No high CPU spike or usage on control or forwarding plane
+    *   No high memory utilization or usage on control or forwarding plane
+    *   No processes/daemons high CPU/Memory utilization
+    *   No generic drop counters
+        *   QUEUE drops
+            *   Interfaces
+            *   VOQ
+    *   ASIC drops
+    *   No flow control frames tx/rx
+    *   No CRC or Layer 1 errors on interfaces
+    *   No config commit errors
+    *   No system level alarms
+    *   In spec hardware should be in proper state
+        *   No hardware errors
+        *   Major Alarms
+    *   No HW component or SW processes crash
+*   TODO:
```

Comment:
Why this is TODO and not implemented?

Reply:
Implemented code for Check No Fabric Drop. For "No CRC or Layer 1 errors fabric links", because there is no oc model path yet, we just keep it in TODO. Is it ok?